### PR TITLE
Setup MongoDB for the GraphQL Server

### PR DESCRIPTION
**Pull request by RwandaGloria**
- ✅Setup mongodb connection with the help of mongoose library
- ✅Use ENVs to securely store the connection URI
- ✅create a database directory under src to store the connection file

### CODE REVIEW

File: packages/server/src/db.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose';
+
+import dotenv from 'dotenv';
+dotenv.config()
+
+const MONGO_STRING = process.env.MONGO_STRING;
+
+// connect to mongodb
+export async function DB_Connect() {
```
Comment (pavankpdev) [inline packages/server/src/db.ts:9]: The usual practice in NodeJS or JavaScript ecosystem is to use camelCase. 
Please rename the function name to `dbConnect()`

File: packages/server/src/db.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    // console.log(process.env.MONGO_STRING);
+    mongoose.connect(process.env.MONGO_STRING!);
+
+    mongoose.connection.on('connected', () => {
+        console.log('Connected to MongoDB');
+    });
+    // console.log(process.env.MONGO_STRING)
```
Comment (pavankpdev) [inline packages/server/src/db.ts:17]: Remove dead code

File: packages/server/src/index.ts [hunk 3]
```diff
@@ -2,18 +2,34 @@ import express from 'express';
...
+
     server.applyMiddleware({ app });
 
-    app.listen({ port: 4000 }, () =>
+    app.listen({ port: 4000 }, async () =>{
+        
+     
+
+        if (!process.env.MONGO_STRING) {
+            console.log("Mongo string is undefined!")
```
Comment (pavankpdev) [inline packages/server/src/index.ts:26]: add `process.exit(1)` if mongo uri is undefined

File: packages/server/src/endpoints.config.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import dotenv from 'dotenv';
```
Comment (pavankpdev) [inline packages/server/src/endpoints.config.ts:1]: What is the purpose of this file?

File: package.json [hunk 5]
```diff
@@ -8,3 +8,21 @@
...
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "devDependencies": {
+    "@types/dotenv": "^8.2.0",
+    "lerna": "^6.6.1"
+  },
+  "dependencies": {
+    "dotenv": "^16.0.3",
```
Comment (pavankpdev) [inline package.json:22]: Seems like the required packages are installed to root, please remove this from here and reinstall them at `packages/server`

File: desktop.ini [hunk 6]
```diff
@@ -0,0 +1,4 @@
+[ViewState]
```
Comment (pavankpdev) [inline desktop.ini:1]: add this file to `.gitignore`

**Review by pavankpdev — CHANGES_REQUESTED** [review]
Please review and update the changes requested. 
In addition to that, I see a few `.js` files inside `Resolver` directory, if they are build files, please make sure not to push them.

File: packages/server/src/db.ts (hunk 1 above)
Comment (RwandaGloria) [reply packages/server/src/db.ts:9]: OK Got it. I will make changes to all you mentioned about.

**Comment by pavankpdev** [comment]
Hey @RwandaGloria mark the reviews as resolved and let me know, once your done.

**Review by pavankpdev — APPROVED** [review]
Good work, merging this PR. 
Although, I might have to setup formatter and a git hooks.

**Pull request merged**